lib/orbit-common/cache-transformer.js

```diff
@@ -0,0 +1,92 @@
+import { eq } from 'orbit/lib/eq';
+import { Class } from 'orbit/lib/objects';
+
+export default Class.extend({
+  init: function(retrieve, transformTarget, schema, processors){
```

Comment:
Perhaps `retrieveFunction` and `transformFunction`? Alternatively, we could require a `target` that must support both `retrieve` and `transform`.